src/utils/workflow-utils.ts

```
@@ -0,0 +1,17 @@
+/**
+ * Workflow utility helpers
+ */
+
+/**
+ * Remove read-only and unsupported properties from a workflow object
+ * before sending it to the n8n API.
+ */
+export function sanitizeWorkflowForApi(workflow: Record<string, any>): Record<string, any> {
+  const sanitized = { ...workflow };
+  delete sanitized.id;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)